2024-01-26 17:34:42,254 INFO    StreamThr :3081773 [internal.py:wandb_internal():86] W&B internal server running at pid: 3081773, started at: 2024-01-26 17:34:42.253874
2024-01-26 17:34:42,299 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status
2024-01-26 17:34:42,307 INFO    WriterThread:3081773 [datastore.py:open_for_write():87] open: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/run-f2an1mca.wandb
2024-01-26 17:34:42,308 DEBUG   SenderThread:3081773 [sender.py:send():382] send: header
2024-01-26 17:34:42,308 DEBUG   SenderThread:3081773 [sender.py:send():382] send: run
2024-01-26 17:34:45,398 INFO    SenderThread:3081773 [dir_watcher.py:__init__():211] watching files in: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files
2024-01-26 17:34:45,398 INFO    SenderThread:3081773 [sender.py:_start_run_threads():1136] run started: f2an1mca with start time 1706261682.214285
2024-01-26 17:34:45,414 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: check_version
2024-01-26 17:34:45,415 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: check_version
2024-01-26 17:34:45,915 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: run_start
2024-01-26 17:34:45,986 DEBUG   HandlerThread:3081773 [system_info.py:__init__():27] System info init
2024-01-26 17:34:45,986 DEBUG   HandlerThread:3081773 [system_info.py:__init__():42] System info init done
2024-01-26 17:34:45,986 INFO    HandlerThread:3081773 [system_monitor.py:start():194] Starting system monitor
2024-01-26 17:34:46,003 INFO    SystemMonitor:3081773 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-01-26 17:34:46,003 INFO    HandlerThread:3081773 [system_monitor.py:probe():214] Collecting system info
2024-01-26 17:34:46,081 INFO    SystemMonitor:3081773 [interfaces.py:start():190] Started cpu monitoring
2024-01-26 17:34:46,121 INFO    SystemMonitor:3081773 [interfaces.py:start():190] Started disk monitoring
2024-01-26 17:34:46,132 INFO    SystemMonitor:3081773 [interfaces.py:start():190] Started gpu monitoring
2024-01-26 17:34:46,145 INFO    SystemMonitor:3081773 [interfaces.py:start():190] Started memory monitoring
2024-01-26 17:34:46,162 INFO    SystemMonitor:3081773 [interfaces.py:start():190] Started network monitoring
2024-01-26 17:34:46,253 DEBUG   HandlerThread:3081773 [system_info.py:probe():151] Probing system
2024-01-26 17:34:46,296 DEBUG   HandlerThread:3081773 [system_info.py:_probe_git():136] Probing git
2024-01-26 17:34:46,341 DEBUG   HandlerThread:3081773 [system_info.py:_probe_git():144] Probing git done
2024-01-26 17:34:46,341 DEBUG   HandlerThread:3081773 [system_info.py:probe():199] Probing system done
2024-01-26 17:34:46,341 DEBUG   HandlerThread:3081773 [system_monitor.py:probe():223] {'os': 'Linux-5.15.0-91-generic-x86_64-with-glibc2.17', 'python': '3.8.17', 'heartbeatAt': '2024-01-26T09:34:46.253329', 'startedAt': '2024-01-26T09:34:42.187487', 'docker': None, 'cuda': None, 'args': ('--local_rank=4', '--config_file', 'config/bert_pretraining_phase2_config.json', '--input_dir', '/home/yanxindong/mkor/bert/data/encoded/sequences_lowercase_max_seq_len_512_next_seq_task_true', '--output_dir', 'results/adam', '--weight_decay', '0.01', '--num_steps_per_checkpoint', '200'), 'state': 'running', 'program': 'run_pretraining.py', 'codePathLocal': 'run_pretraining.py', 'codePath': 'bert/run_pretraining.py', 'git': {'remote': 'https://github.com/kkkjj-d/mkor.git', 'commit': 'efe99e84fef5594780cfb1df6d6a1f61da4ebba0'}, 'email': '1262670315@qq.com', 'root': '/home/yanxindong/mkor', 'host': 'a6000-X640-G40', 'username': 'yanxindong', 'executable': '/home/yanxindong/anaconda/envs/eva/bin/python', 'cpu_count': 64, 'cpu_count_logical': 128, 'cpu_freq': {'current': 1835.5633984375, 'min': 800.0, 'max': 3400.0}, 'cpu_freq_per_core': [{'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 2000.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.003, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3299.996, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 1800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 1100.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3100.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3299.997, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 1900.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2000.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 1800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 2000.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 2500.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 2202.814, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 1100.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 1000.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}], 'disk': {'/': {'total': 438.05088806152344, 'used': 28.072593688964844}}, 'gpu': 'NVIDIA RTX A6000', 'gpu_count': 8, 'gpu_devices': [{'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}], 'memory': {'total': 503.51250076293945}}
2024-01-26 17:34:46,341 INFO    HandlerThread:3081773 [system_monitor.py:probe():224] Finished collecting system info
2024-01-26 17:34:46,341 INFO    HandlerThread:3081773 [system_monitor.py:probe():227] Publishing system info
2024-01-26 17:34:46,341 DEBUG   HandlerThread:3081773 [system_info.py:_save_conda():208] Saving list of conda packages installed into the current environment
2024-01-26 17:34:46,394 INFO    Thread-12 :3081773 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files/conda-environment.yaml
2024-01-26 17:34:56,395 INFO    Thread-12 :3081773 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files/conda-environment.yaml
2024-01-26 17:34:56,701 DEBUG   HandlerThread:3081773 [system_info.py:_save_conda():220] Saving conda packages done
2024-01-26 17:34:56,703 INFO    HandlerThread:3081773 [system_monitor.py:probe():229] Finished publishing system info
2024-01-26 17:34:56,709 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:34:56,715 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:34:56,715 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:34:56,715 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:34:56,715 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: python_packages
2024-01-26 17:34:56,715 DEBUG   SenderThread:3081773 [sender.py:send():382] send: files
2024-01-26 17:34:56,716 INFO    SenderThread:3081773 [sender.py:_save_file():1403] saving file wandb-metadata.json with policy now
2024-01-26 17:34:56,716 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: python_packages
2024-01-26 17:34:56,730 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:34:56,730 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:34:56,764 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:34:57,078 DEBUG   SenderThread:3081773 [sender.py:send():382] send: telemetry
2024-01-26 17:34:57,395 INFO    Thread-12 :3081773 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files/requirements.txt
2024-01-26 17:34:57,396 INFO    Thread-12 :3081773 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files/wandb-metadata.json
2024-01-26 17:34:57,698 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80eea30>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:34:57,698 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15620'}
2024-01-26 17:34:57,698 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:34:58,079 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:34:58,882 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80ee040>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:34:58,882 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15620'}
2024-01-26 17:34:58,882 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:35:01,322 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80f8310>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:35:01,322 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15620'}
2024-01-26 17:35:01,322 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:35:01,322 INFO    wandb-upload_0:3081773 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fa2c80f8310>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 799, in urlopen
    retries = retries.increment(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80f8310>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2758, in upload_file
    response = self._upload_file_session.put(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80f8310>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2789, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2758, in upload_file
    response = self._upload_file_session.put(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-01-26 17:35:03,079 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:06,315 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80f8bb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:35:06,315 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15620'}
2024-01-26 17:35:06,315 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:35:08,081 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:11,730 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:35:11,730 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:35:11,774 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:35:14,791 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:15,406 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c07c0a30>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:35:15,406 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15620'}
2024-01-26 17:35:15,406 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:35:17,405 INFO    Thread-12 :3081773 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files/config.yaml
2024-01-26 17:35:21,598 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:26,632 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:26,730 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:35:26,731 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:35:26,774 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:35:32,106 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:32,175 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80f8a60>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:35:32,175 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15620'}
2024-01-26 17:35:32,175 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:35:37,106 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:41,730 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:35:41,730 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:35:41,774 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:35:43,023 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:46,165 DEBUG   SystemMonitor:3081773 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-01-26 17:35:46,169 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:35:48,170 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:53,171 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:35:56,731 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:35:56,731 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:35:56,774 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:35:59,058 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:04,094 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:07,102 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/f2an1mca/wandb-metadata.json?Expires=1706348097&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=PAhJqlZYgyrxW9zQN%2FulJBQzry63c5KvWoE40ziH7734xV8slJ5KOlhlwKW%2Bv1VFghH4n4jVsl%2BRNop7b1UDv5MR82%2Bb7hr6%2F74DhVaCX0ByL4Q8IGrkRFSBXNIQEnu1M4JCxxuk2%2F93X3eWOp6iYFcLj1td0rBroCstsUxQxAkS2iPMuN2fjlvCtn8qCSSTGXyGHn6TEbpuWODUBx%2BepZmMBYPf%2BiCZpwsKqo0zIGmT68%2FTgjU8DFl%2B5k4fvm4keiYwbFhKweotu71WOamGcZXAWBZKBVKxRPf2gArhGxNVDIRZwUjt5aO10IWLfUGV%2BTMwb8GhvJA99651z%2BpibQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2c80f81f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:36:07,103 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15620'}
2024-01-26 17:36:07,103 ERROR   wandb-upload_0:3081773 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:36:09,105 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:11,742 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:36:11,742 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:36:11,774 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:36:15,049 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:16,172 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:36:20,174 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:25,174 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:26,730 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:36:26,731 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:36:26,774 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:36:31,001 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:32,556 INFO    Thread-12 :3081773 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files/output.log
2024-01-26 17:36:34,556 INFO    Thread-12 :3081773 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/files/output.log
2024-01-26 17:36:36,173 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:41,173 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:42,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:36:42,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:36:42,658 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:36:46,173 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:36:46,176 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:51,514 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:56,817 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:36:57,656 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:36:57,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:36:57,658 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:37:02,071 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:07,169 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:12,169 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:12,656 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:37:12,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:37:12,658 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:37:16,144 INFO    wandb-upload_0:3081773 [upload_job.py:push():131] Uploaded file /tmp/tmpd72e9jodwandb/n7v6x2vk-wandb-metadata.json
2024-01-26 17:37:16,177 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:37:17,223 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:22,257 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:27,570 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:27,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:37:27,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:37:27,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:37:33,095 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:38,097 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:42,656 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:37:42,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:37:42,658 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:37:43,983 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:46,179 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:37:49,180 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:54,181 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:37:57,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:37:57,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:37:57,658 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:38:00,030 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:05,112 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:10,154 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:12,706 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:38:12,706 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:38:12,712 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:38:16,044 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:16,182 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:38:21,262 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:26,264 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:27,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:38:27,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:38:27,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:38:32,049 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:37,110 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:42,169 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:42,686 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:38:42,686 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:38:42,686 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:38:46,185 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:38:47,187 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:52,187 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:57,188 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:38:57,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:38:57,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:38:57,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:39:02,947 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:07,948 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:12,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:39:12,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:39:12,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:39:13,000 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:16,187 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:39:18,188 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:23,320 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:27,718 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:39:27,718 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:39:27,719 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:39:29,105 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:34,165 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:39,364 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:42,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:39:42,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:39:42,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:39:45,062 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:46,189 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:39:50,316 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:55,597 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:39:57,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:39:57,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:39:57,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:40:00,932 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:05,932 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:10,933 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:12,657 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:40:12,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:40:12,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:40:16,191 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:40:16,195 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:21,193 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:26,194 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:27,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:40:27,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:40:27,659 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:40:32,034 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:37,172 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:42,187 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:42,658 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:40:42,678 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:40:42,678 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:40:46,193 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:40:47,230 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:52,419 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:57,487 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:40:57,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:40:57,662 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:40:57,662 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:41:03,027 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:08,028 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:12,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:41:12,660 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:41:12,660 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:41:14,026 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:16,196 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:41:19,198 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:24,198 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:27,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:41:27,660 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:41:27,661 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:41:29,966 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:35,013 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:40,058 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:42,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:41:42,661 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:41:42,661 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:41:45,967 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:46,198 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:41:51,298 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:56,369 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:41:57,693 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:41:57,694 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:41:57,694 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:42:02,021 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:07,021 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:12,022 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:12,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:42:12,661 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:42:12,661 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:42:16,200 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:42:17,202 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:22,202 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:27,203 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:27,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:42:27,661 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:42:27,661 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:42:33,032 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:38,033 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:42,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:42:42,661 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:42:42,661 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:42:44,022 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:46,202 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:42:49,239 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:54,333 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:42:57,659 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:42:57,661 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:42:57,661 DEBUG   SenderThread:3081773 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:43:00,018 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:43:05,019 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:43:09,889 INFO    cpu       :3081773 [interfaces.py:monitor():140] Process cpu has exited.
2024-01-26 17:43:09,892 DEBUG   SystemMonitor:3081773 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-01-26 17:43:09,892 DEBUG   SystemMonitor:3081773 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-01-26 17:43:09,895 DEBUG   SenderThread:3081773 [sender.py:send():382] send: stats
2024-01-26 17:43:10,048 INFO    MainThread:3081773 [internal.py:handle_exit():76] Internal process exited
2024-01-26 17:43:10,896 DEBUG   HandlerThread:3081773 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:43:11,061 WARNING StreamThr :3081773 [internal.py:is_dead():414] Internal process exiting, parent pid 3081501 disappeared
2024-01-26 17:43:11,061 ERROR   StreamThr :3081773 [internal.py:wandb_internal():152] Internal process shutdown.
2024-01-26 17:43:11,896 INFO    SenderThread:3081773 [sender.py:finish():1572] shutting down sender
2024-01-26 17:43:11,897 INFO    HandlerThread:3081773 [handler.py:finish():869] shutting down handler
2024-01-26 17:43:11,897 INFO    HandlerThread:3081773 [system_monitor.py:finish():203] Stopping system monitor
2024-01-26 17:43:11,897 INFO    HandlerThread:3081773 [interfaces.py:finish():202] Joined cpu monitor
2024-01-26 17:43:11,897 INFO    HandlerThread:3081773 [interfaces.py:finish():202] Joined disk monitor
2024-01-26 17:43:11,897 INFO    HandlerThread:3081773 [interfaces.py:finish():202] Joined gpu monitor
2024-01-26 17:43:11,897 INFO    HandlerThread:3081773 [interfaces.py:finish():202] Joined memory monitor
2024-01-26 17:43:11,897 INFO    HandlerThread:3081773 [interfaces.py:finish():202] Joined network monitor
2024-01-26 17:43:11,897 INFO    WriterThread:3081773 [datastore.py:close():296] close: /home/yanxindong/mkor/bert/wandb/run-20240126_173442-f2an1mca/run-f2an1mca.wandb
